src/js/components/Select/Select.js

```diff
@@ -23,7 +23,7 @@ import { SelectContainer } from './SelectContainer';
 import { applyKey } from './utils';
 
 const SelectTextInput = styled(TextInput)`
-  cursor: pointer;
+  cursor: ${props => (props.defaultCursor ? 'default' : 'pointer')};
```

Comment:
why not using `props.disabled` directly and remove the `defaultCursor` prop?

Reply:
If we pass as `disabled` then `opacity` will be affected as well. We don't want opacity to be lowered by both Select and the underlying TextInput. We just want to control the cursor.